Templates/Files/Local Files/Azure Document Intelligence - Extract data and save results to database/long-description.md

```diff
@@ -0,0 +1,24 @@
+This template reads files from a local fileshare, extracts data from them with Azure Document Intelligence, and inserts them into a Microsoft SQL database table.
+
+![Template](assets/Azure_Document_Intelligence_-_Extract_data_and_save_results_to_database.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Frends agent has access to the local directory containing the files.
+- The Azure user has the required API key and endpoint URL to use the Document Intelligence API.
+- The Microsoft SQL user has access to the database with permissions to perform the insert.
+- The database table where the results will be saved needs to be created beforehand.
```

Comment:
Please mention the database table format that is used by the template and add the database CREATE TABLE statement, so that users can easily reproduce the results.